hedera-mirror-rest/topicmessage.js

```diff
@@ -400,6 +409,39 @@ const getMessages = async (pgSqlQuery, pgSqlParams, preQueryHint) => {
   return rows.map((row) => new TopicMessage(row));
 };
 
+const getQueryWithTopicMessageLookupTimestamps = async (topicId, seqNum) => {
+  // Get the timestamp range from the topic message lookup table
+  const sequenceNumber = BigInt(seqNum);
+  const topicMessageLookupQuery = `select lower(timestamp_range) as timestamp_start, upper(timestamp_range) as timestamp_end
```

Comment:
```suggestion
  const topicMessageLookupQuery = `select timestamp_range
```

and the postgres driver will decode it into a `pg-range.Range` object